library-conan/conanfile.py

```diff
@@ -0,0 +1,69 @@
+import os
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import update_conandata
+
+class DPPConan(ConanFile):
+    name = "dpp"
+    version = "10.0.29"
+    default_user = "brainboxdotcc"
+    default_channel = "testing"
+    license = "https://github.com/brainboxdotcc/DPP/blob/master/LICENSE"
+    url = "https://github.com/brainboxdotcc/DPP"
+    description = "D++ is a lightweight and efficient library for Discord"
+    topics = ("discord")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def build_requirements(self):
```

Comment:
```suggestion
    def requirements(self):
```
those are regular ``requires``, recommended to go on the ``requirements()`` method